src/main/webapp/app/lecture/pdf-preview/pdf-preview.component.ts

```diff
@@ -0,0 +1,203 @@
+import { Component, ElementRef, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AttachmentService } from 'app/lecture/attachment.service';
+import * as PDFJS from 'pdfjs-dist';
+import 'pdfjs-dist/build/pdf.worker';
```

Comment:
Just checking: We use the fake worker?

Reply:
Yes, actually I managed to add the real worker on my local but on the prod version at first it gave me this error:

Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.

**Which I added to WebConfigurer as:**
mappings.add("mis", MediaType.TEXT_HTML_VALUE + ";charset=" + StandardCharsets.UTF_8.name().toLowerCase());

**Then I got this error:** 
Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.

mappings.add("text/javascript", MediaType.TEXT_HTML_VALUE + ";charset=" + StandardCharsets.UTF_8.name().toLowerCase());

But it didn't fix the last issue. Any ideas?

Reply:
Maybe

```
mappings.add("mjs", "text/javascript;charset=" + StandardCharsets.UTF_8.name().toLowerCase());
```

would fix it? (It's `mjs`, not `mis`, right?)

Reply:
No sorry that was auto correction of my keyboard, I used mjs. And that was working, but couldn't fix this one after setting mjs:

`Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
`

Reply:
This is probably not super urgent for now. Let's maybe look at this in a follow-up PR since the fake worker seems to work fine

Reply:
Sure